### PR TITLE
fix inconsistency

### PR DESCRIPTION
**Pull request by ahkuma**

### CODE REVIEW

File: src/templates/static/get_started/binary_options.jsx [hunk 1]
```diff
@@ -236,12 +236,12 @@ const BinaryOptions = () => {
                             <ListStrong header={it.L('Even/Odd')}        text={it.L('Predict that the last digit is an even number or odd number after the last tick')} />
                             <ListStrong header={it.L('Over/Under')}      text={it.L('Predict that the last digit is higher or lower')} />
                         </SectionLarge>
-                        <SectionLarge dataAnchor='reset-callput' header={it.L('Reset Call/Put')} image='types-of-trades/reset' text={it.L('There are two types of reset trades:')}>
+                        <SectionLarge dataAnchor='reset-callput' header={it.L('Reset Call/Reset Put')} image='types-of-trades/reset' text={it.L('There are two types of reset trades:')}>
                             <ListStrong header={it.L('Reset-Call')} text={it.L('Predict that the market will end up higher than either the current level or the level at a predetermined time (reset time)')} />
```
Comment (kellybinary) [inline src/templates/static/get_started/binary_options.jsx]: should this be changed to Reset Call (without the dash)?

File: src/templates/static/get_started/binary_options.jsx [hunk 2]
```diff
@@ -236,12 +236,12 @@ const BinaryOptions = () => {
...
                             <ListStrong header={it.L('Reset-Put')}  text={it.L('Predict that the market will end up lower than either the current level or the level at a predetermined time (reset time)')} />
```
Comment (kellybinary) [inline src/templates/static/get_started/binary_options.jsx]: ditto: Reset Put

File: src/templates/static/get_started/binary_options.jsx (hunk 1 above)
Comment (ahkuma) [reply src/templates/static/get_started/binary_options.jsx]: Yep, Dervin specifically told us to use that (without dash)

Comment (ahkuma) [reply src/templates/static/get_started/binary_options.jsx]: https://binarygroup.mojohelpdesk.com/ma/#/tickets/21983121

Comment (kellybinary) [reply src/templates/static/get_started/binary_options.jsx]: @ahkuma line 240 needs to be changed to **Reset Call** and line 241 to **Reset Put**😄

**Pull request merged**